keyboards/fjlabs/bolsa65/info.json

```diff
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "Bolsa65",
+    "url": "https://www.bolsakeyboardsupply.com",
+    "maintainer": "FJLabs",
+    "width": 15,
```

Comment:
I missed a small thing.

```suggestion
    "width": 16,
```

Reply:
Ah, thought that was total matrix col's, not U width, thanks.